src/etc/inc/upgrade_config.inc

```diff
@@ -5063,4 +5063,88 @@ function upgrade_159_to_160() {
 		}
 	}
 }
+
+/* RAM Disk Management */
+function upgrade_160_to_161() {
+	global $g, $config;
+
+	if (isset($config['system']['use_mfs_tmpvar'])) {
+
+		// Move existing RRD backup to the RAM Disk Store if it don't already exist there.
+		// Restore existing RRD XML dump backup.
+		if (file_exists("{$g['cf_conf_path']}/rrd.tgz") && !file_exists("{$g['cf_conf_path']}/RAM_Disk_Store/rrd.tgz")) {
+			$rrddbpath = "{$g['vardb_path']}/rrd/";
+			$rrdtool = "/usr/bin/nice -n20 /usr/local/bin/rrdtool";
+
+			$rrdrestore = "";
+			$rrdreturn = "";
+			foreach (glob("{$rrddbpath}/*.xml") as $xml_file) {
+				@unlink($xml_file);
+			}
+			unset($rrdrestore);
+			$_gb = exec("cd /;LANG=C /usr/bin/tar -tf {$g['cf_conf_path']}/rrd.tgz", $rrdrestore, $rrdreturn);
```

Comment:
cd / can be removed, tar -tf is a read-only operation that will only print filenames